binary_search/binary_search.c

```diff
@@ -5,6 +5,39 @@
  * @arr_size Size of array
  * @search_element Element to be searched
  */
+ 
+/* function to perform recursive binary search
+ * input parameters:
+ * @ar - user-input array.
+ * @l - left index of array.
+ * @r - right index of array.
+ * @ele - element to be searched.
+ */
+int recursive_binary_search(const int *ar, int l, int r, int ele) {
+	if (r >= l) {
+		
+		int mid = l + (r - l) / 2;    //calculate mid point of the array
+		
+		// If the element is present at the middle itself
+		if (ar[mid] == ele) {  
+			return mid;   
+		}
+		
```

Comment:
Just undo what change is show in binary_search.c file https://github.com/iiitv/algos/pull/253/files#diff-2b5f8e5dede4db6a4fea7b2769c4f983R8